source/concepts/protocols.rst

```diff
@@ -0,0 +1,309 @@
+.. SPDX-FileCopyrightText: 2021 Hyperledger
+   SPDX-License-Identifier: CC-BY-4.0
+
+*********
+Protocols
+*********
+
+Perun state channel protocols are a set of protocols for setting up a state
+channels, doing off-chain transactions on these channels and settling them.
+
+The life cycle of a state channel consists of 4 phases:
+
+1. **Open phase**: Setup up the channel and deposit funds.
+2. **Transact phase**: Do off-chain transactions.
+3. **Register phase**: Agree on what state should be settled.
+4. **Settle phase**: Settle the channel and withdraw funds.
```

Comment:
Thought about it another time. An alternative closer to your original proposal is:

2. Transact: Off-chain transactions. (without off-chain finalization)
3. Finalize: Finalize off-chain, on dispute, finalize on-chain. If app channel, perform on-chain progression. Conclude the channel.
4. Withdraw: Withdraw the funds.

Reply:
`Finalize` sounds more descriptive than `Register`. Hence, I have accepted it and revised the content and diagrams accordingly. I also like the idea to remove "off-chain finalization" from Transact phase.

However, I would still like to call the last phase as `Settle` instead of `Withdraw`, because this phase involves two steps: Conclude and Withdraw. You can see the updated content in the section on  `Settle` phase for more details.